src/lib.rs

```diff
@@ -43,21 +43,21 @@
 //!     // Get a TOTP instance with an '&str' secret and default SHA1 Digest
 //!     let totp_sha1_str = TOTP::from_utf8(secret);
 //!     // Get an otp with the given counter and elapsed seconds
-//!     let otp_sha1 = totp_sha1_str.get_otp(elapsed_seconds, 8);
+//!     let otp_sha1 = totp_sha1_str.get_otp(elapsed_seconds);
 //!     println!("The otp from totp_sha1_str: {}", otp_sha1);
 //!
 //!     // Alternatively get a TOTP instance with an '&[u8]' secret
 //!     // and different digest (Sha256 or Sha512)
-//!     let totp_sha256_bytes = TOTP::new_with_digest(
+//!     let totp_sha256_bytes = TOTP::new(
 //!         secret.as_bytes(),
-//!         MacDigest::SHA256
+//!         MacDigest::SHA256, // SHA256 algorithm
+//!         8,  // 8 digits
+//!         60  // 60 seconds interval
```

Comment:
Does "60-second interval" sound better? instead of "60 seconds interval"

Reply:
I don't know, I'm not an English native speaker (I'm French) so my English may sound a bit weird.

Reply:
ah okay, yeah can we switch it over to "60-second interval" then?

Reply:
Already did it in latest commit

Reply:
Just saw it! Sorry about that